<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseResultFileType</class>
 <widget class="QWidget" name="ChooseResultFileType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选取结果文件类型</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>80</number>
   </property>
   <property name="topMargin">
    <number>40</number>
   </property>
   <property name="bottomMargin">
    <number>40</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="Btn_d3plot">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>D3plot</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Btn_rwforc">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>rwforc</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Btn_multiFile">
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string>MultiFile</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
